src/storage/BaseProcessor.inl

```diff
@@ -67,5 +67,84 @@ void BaseProcessor<RESP>::doPut(GraphSpaceID spaceId,
     });
 }
 
+template<typename RESP>
+void BaseProcessor<RESP>::doRemove(GraphSpaceID spaceId,
+                                   PartitionID partId,
+                                   std::vector<std::string> keys) {
+    this->kvstore_->asyncMultiRemove(spaceId,
+                                     partId,
+                                     std::move(keys),
+                                     [spaceId, partId, this](kvstore::ResultCode code) {
+        VLOG(3) << "partId:" << partId << ", code:" << static_cast<int32_t>(code);
+
+        cpp2::ResultCode thriftResult;
+        thriftResult.set_code(to(code));
+        thriftResult.set_part_id(partId);
+        if (code == kvstore::ResultCode::ERR_LEADER_CHANGED) {
+            nebula::cpp2::HostAddr leader;
+            auto addrRet = kvstore_->partLeader(spaceId, partId);
+            CHECK(ok(addrRet));
```

Comment:
maybe should no use CHECK any more

Reply:
Why ?

Reply:
The check will make process crash directly, and crash is a dangrous behavior that will lost data.

Reply:
Sounds good, this operation also exists elsewhere and can be handled in a new PR.